datapipe.c

```diff
@@ -1017,6 +1017,71 @@ const char *device_lock_state_repr(device_lock_state_t state)
 	return res;
 }
 
+/** Convert uiexctype_t enum to human readable string
+ *
+ * Note that while uiexctype_t actually is a bitmask
+ * and this function is expected to be used in situations
+ * where at maximum one bit is set.
+ *
+ * @param state uiexctype_t exception stata
+ *
+ * @return human readable representation of state
+ */
+const char *uiexctype_repr(uiexctype_t state)
+{
+	const char *res = "unknown";
+	switch( state ) {
+	case UIEXC_NONE:   res = "none";   break;
```

Comment:
How about replacing hardcoded constants with ```MCE_BLANKING_POLICY_*``` constants introduced in https://github.com/nemomobile/mce-dev/pull/10 ?